test/conformance/conformance_blocked_nd_range.cpp

```diff
@@ -291,3 +291,71 @@ TEST_CASE("blocked_nd_range proportional splitting") {
         utils::check_range_bounds_after_splitting(original.dim(0), first.dim(0), second.dim(0), expected_first_end);
     }
 }
+
+#if __TBB_CPP17_DEDUCTION_GUIDES_PRESENT
+//! Testing blocked_rangeNd deduction guides
+//! \brief \ref interface
+TEST_CASE("blocked_rangeNd deduction guides") {
```

Comment:
The test misses any checks for constructing explicitly from an array. It needs to check construction from arrays of different sizes (incl. 2 and 3), with and without a grainsize argument.

Reply:
Done